tox/session.py

```diff
@@ -242,7 +242,7 @@ def __init__(self, session):
     @property
     def verbosity(self):
         if self.session:
-            return self.session.config.option.verbosity
+            return self.session.config.option.verbosity - self.session.config.option.quiet
```

Comment:
First, here I would prefer we rename the verbosity of ``self.session.config.option.verbosity`` to something like ``verbose_level``, to not conflict the property concept. Also rename ``quiet`` to ``quiet_level``, this would make the two options similarly named, and clearly delimiting from the verbosity concept here.